File: Sorry.cs
using System;

namespace HelloWorld
{
  class Program
  {
    static void Main(string[] args)
    {
      bool SvlkEaten = false;
      Console.WriteLine("Did I eat your souvlaki (y/n):");
      string UsrIn = Console.ReadLine();
      switch(UsrIn)
      {
      	case "y":
        case "yes":
          Console.WriteLine("I am terribly sorry");
          break;
          
        case "n":
        case "no":
          Console.WriteLine("Ok");
          break;
      }
    }
  }
}

File: Sorry.c
#include <stdio.h>
#include <string.h>

int main() {
    char UsrIn[3];
    printf("Did I eat your souvlaki (y/n):");
    scanf("%c", UsrIn);
    if( strcmp (UsrIn, "y") == 0 || strcmp (UsrIn, "yes") == 0)
    {
        printf("I am terribly sorry");
    }
    if(strcmp (UsrIn, "n") == 0|| strcmp (UsrIn, "no") == 0)
    {
        printf("ok");
    }
    return 0;
}
